templates/components/checkbox.build.hbs

```diff
@@ -1,8 +1,19 @@
-<template v-for="(option, index) in options">
-  <div class="checkbox checkbox-icon">
-    <input type="checkbox" :id="name + '-' + index" :name="name" v-model="value" :value="option.id" :required="isRequired" v-on:change="updateValue()">
-    <label :for="name + '-' + index">
-      <span class="check"><i class="fa fa-check"></i></span> \{{ option.label || option.id }}
-    </label>
-  </div>
+<template>
+    <template v-for="(option, index) in options">
+        <div class="checkbox checkbox-icon">
+            <input
+                    type="checkbox"
+                    :id="name + '-' + index"
+                    :name="name"
+                    v-model="value"
+                    :value="option.label || option.id"
+                    v-on:change="updateValue()"
```

Comment:
`>` needs to be on the same line as the last property:

```
<input foo
     bar
     baz>
```

Reply:
fixed